app/com/arpnetworking/metrics/portal/reports/ReportSpec.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2018 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.metrics.portal.reports;
+
+import java.util.concurrent.CompletionStage;
+
+/**
+ * Describes how to render a report.
```

Comment:
Not really; I mean this doesn't just describe how to do it, it actually does it. Right?